doc/source/admin/framework_dependencies.rst

```diff
@@ -241,38 +226,20 @@ dependencies directly from PyPI_.
 Adding additional Galaxy dependencies
 -------------------------------------
 
-New packages can be added to Galaxy, or the versions of existing packages can be updated, using `pipenv`_ and `Starforge`_, Galaxy's Docker-based build system.
+New packages can be added to Galaxy, or the versions of existing packages can be updated, using `poetry`_ and `Starforge Recipes`_, Galaxy's Docker-based build system.
 
 The process is still under development and will be streamlined and automated over time. For the time being, please use
 the following process to add new packages and have their wheels built:
 
-1. Install `Starforge`_ (e.g. with ``pip install starforge`` or ``python setup.py install`` from the source). You will
-   also need to have Docker installed on your system.
+1. Clone https://github.com/galaxyproject/starforge-recipes/ and add or edit the wheel you would like to build under the wheels/ directory.
```

Comment:
This is needed only if the package is not pure Python and is not already available on PyPI (for one of the platforms or Python versions supported by Galaxy).